src/main/java/gov/nih/nci/icdc/IcdcEsFilter.java

```diff
@@ -269,7 +274,11 @@ private Map<String, Object> searchCases(Map<String, Object> params) throws IOExc
         Request studyFileCountRequest = new Request("GET", FILES_COUNT_END_POINT);
         Map<String, Object> studyFileParam = new HashMap<>(formattedParams);
         studyFileParam.put("file_level", List.of("study"));
-        Map<String, Object> studyFileQuery = esService.buildFacetFilterQuery(studyFileParam, Set.of(), Set.of("first"));
+        Map<String, Object> studyFileQuery = esService.buildFacetFilterQuery(studyFileParam, Set.of(), Set.of("first", SEARCH_TEXT));
+        if (!studyFileParam.get(SEARCH_TEXT).toString().isEmpty()) {
```

Comment:
if `studyFileParam.get(SEARCH_TEXT)` is null this code will throw an error. It would probably be safer to add a try catch block around this.

Reply:
I think this change avoids the error altogether - let me know what you think.